3rdparty/osqp/CMakeLists.txt

```
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(osqp)
+
+include(ExternalProject)
+
+ExternalProject_Add(
+  osqp
+  GIT_REPOSITORY https://github.com/oxfordcontrol/osqp
+  GIT_TAG v0.6.0
+  GIT_SHALLOW     TRUE
```

```suggestion
  URL https://github.com/osqp/osqp/releases/download/v0.6.2/complete_sources.tar.gz
  URL_HASH MD5=66738e622174eb506e0cee106891397e
```

1. https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/442#issuecomment-1461110583 のコメントに従い、`tar.gz`をダウンロードする変更です. git submoduleがある関係でhttps://github.com/osqp/osqp/archive/refs/tags/v0.6.0.tar.gz をダウンロードするだけではソースが足りないため、https://github.com/osqp/osqp/releases/download/v0.6.2/complete_sources.tar.gz をダウンロードしています
2. OSQPのv0.6.2がリリースされているため( https://github.com/osqp/osqp/releases ) v0.6.0ではなくv0.6.2を使うようにしたいのですが、よろしいでしょうか。

---

Thanks! I have corrected based on your suggestion. see 41f4265 .